library/src/main/java/kwasm/format/text/Type.kt

```diff
@@ -62,25 +69,55 @@ sealed class Type(
     class Result(
         sequence: CharSequence,
         context: ParseContext? = null
-    ) : Type(sequence, context) {
+    ) : Type<Unit>(sequence, context) {
         override fun parseValue() {
             TODO("not implemented")
         }
     }
 
+    /**
+     * ```
```

Comment:
Nit: Please provide a link to the part of the spec where the grammar comes from for this.

Reply:
Done